spec/support/self_service/firms_index_page.rb

```diff
@@ -1,13 +1,15 @@
 module SelfService
   class FirmsIndexPage < SitePrism::Page
     set_url '/self_service/firms'
-    set_url_matcher %r{/self_service/firms}
+    set_url_matcher %r{/self_service}
```

Comment:
yeah, in the feature tests the user logs in and lands at `/self_service` which now displays the `firms_index_page`